### ISSUE TITLE
Duplicated components in dependency graph

### ISSUE THREAD

**Issue by marcelpinto** [dvdciri/daggraph#45]
### Description
Duplicated components in graphs (all of them). Seems that the modules get duplicated and appear several times.
### Steps to reproduce
Just run daggraph in my project. Is a multi-module project (4 in total). In each of the modules, there are Dagger DI related files.
### Expected behavior
Component shows only once.

### Screenshot
![image](https://user-images.githubusercontent.com/2639977/38407166-b291e530-3978-11e8-9653-cc8726702a06.png)

**Comment by tharoldD** [comment]
Are you using, Kotlin? Just transitioned to Kotlin and the graph got duplicated like yours.
Still had the java codebase lying around and ran the digraph command in the java project, no duplicates.
Could this be happening only in Kotlin projects?

**Comment by marcelpinto** [comment]
yes, the project is only kotlin.

**Comment by elevenetc** [comment]
The same issue. Would be useful if `daggraph` has some debug mode with logs while it traversing classes.

**Comment by dvdciri** [comment]
Correct, this was a known issue and i just dropped a commit in develop to fix this.

The reason for this is that during components analysis, i was only discarding the "build/generated" folder while in the "build/tmp" folder we can find some components file that the library was using.

Can you guys please do a quick test with the latest develop and let me know (just to double check) that the issue is 100% solved?
@skimarxall is your project open source? If so can you share the link so i can test it out?

**Comment by dvdciri** [comment]
@elevenetc What you mentioned is something that i wanted to do for long time, never actually had time to do it. Can you please open a new issue specific to that so someone can work on it.
Thanks

**Comment by dvdciri** [comment]
0.3.2 is out and should have fixed the problem. Closing this for now, will reopen if needed.
